### PR TITLE
Feat/add workspace

### PR DESCRIPTION
**Pull request by montakarlo**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "eslint-plugin-react-hooks": "4.0.8",
     "history": "5.0.0",
     "node-sass": "4.14.1",
+    "prop-types": "^15.7.2",
```
Comment (AlexanderTirik) [inline frontend/package.json:25]:
```suggestion
    "prop-types": "15.7.2",
```

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  font-family: 'Recursive'
+  width: 100%
+  height: 100vh
+  display: flex
+  justify-content: center
+  align-items: center
+  &__container
+    width: 1024px
+    height: 75vh
+    // background-color: red
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.module.sass:15]: Remove commented code

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass [hunk 3]
```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap')
+@import url('https://fonts.googleapis.com/css2?family=Recursive&display=swap')
+@import url('https://fonts.googleapis.com/css2?family=Recursive:wght@400;600;800&display=swap')
+
+.workspacePage
+  font-family: 'Recursive'
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.module.sass:6]: General font should be set globally

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+.inputContainer
+  width: 80%
+  &__header
+    font-size: 22px
+    letter-spacing: 0px
+    line-height: 1.2
+    margin-bottom: 12px
+    font-weight: 600
+
+.inputButton
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.module.sass:48]: Styles should be set with component itself

File: frontend/src/components/WorkspacePage/WorkspacePage.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import { Form, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import styles from './WorkspacePage.module.sass';
+import logo from '../../img/chatitoTemp.png';
+import bgImg from '../../img/bg-img.png';
+import { connect, useDispatch } from 'react-redux';
+import addWorkspace from '../../reducers/actions/actions';
+
+// const dispatch = useDispatch();
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.tsx:11]: Remove

File: frontend/src/components/WorkspacePage/WorkspacePage.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    console.log('sth');
+  };
+
+  const [name, setState] = useState('');
+  return (
+    <div className={styles.workspacePage}>
+      <div className={styles.workspacePage__container}>
+        <Form className={styles.inputWorkspace} onSubmit={onSubmit}>
+          <div className={styles.logoContainer}>
+            <img className={styles.logoContainer__item} src={logo} alt="logo" />
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.tsx:28]: Bootstrap Image component?

File: frontend/src/components/WorkspacePage/WorkspacePage.tsx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+          <div className={styles.infoSide__imageContainer}>
+            <img src={bgImg} alt="background" />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+// WorkspacePage.propTypes = {
```
Comment (vlvereta) [inline frontend/src/components/WorkspacePage/WorkspacePage.tsx:63]: Not sure that we need it as we're using ts

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass [hunk 8]
```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap')
```
Comment (AlikRakhmonov) [inline frontend/src/components/WorkspacePage/WorkspacePage.module.sass:1]: rename WokspacePage.module.sass to styles.module.sass

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+    height: 100%
+    display: flex
+    justify-content: center
+    align-items: center
+    &>img
+      width: 60%
+  
+.infoButton
+  min-width: 165px
+  margin: 0 0 0 15px
```
Comment (AlikRakhmonov) [inline frontend/src/components/WorkspacePage/WorkspacePage.module.sass:85]: no new line

File: frontend/src/reducers/actions/actions.ts [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import { ADD_WORKSPACE } from './actionTypes';
+
+export default function addWorkspace(name: string) {
```
Comment (AlikRakhmonov) [inline frontend/src/reducers/actions/actions.ts:3]: do not use actions. Use routines

File: frontend/src/components/WorkspacePage/WorkspacePage.module.sass (hunk 8 above)
Comment (AlikRakhmonov) [reply frontend/src/components/WorkspacePage/WorkspacePage.module.sass:1]: include fonts in index.html

**Pull request closed without merging**